components/status-message/assets/StatusMessage.scss

```diff
@@ -0,0 +1,54 @@
+@import "@hods/sass-base";
+@import "govuk-frontend/govuk/components/button";
+
+$govuk-button-colour: govuk-colour("white");
+$govuk-button-text-colour: govuk-colour("black");
+
+.hods-status-message {
+  color: govuk-colour("white");
+  background-color: govuk-colour("blue");
+  @include govuk-responsive-padding(4, "all");
+
+  &__extra {
+    font-weight: normal;
+  }
+
+  &__actions a {
```

Comment:
I wonder if this should re-style all links or just buttons?

Reply:
This is only applied to links in the `actions` part of the component, which is a very specialised and small part of the UI, I'm pretty happy for this to stay as it is.

Reply:
Depending on what you do elsewhere (with the interface), I suspect this should become `&__button` or something.

Reply:
Adopted this style in latest commit